payment/src/main/resources/application.yml

```diff
@@ -0,0 +1,20 @@
+spring:
+  application:
+    name: payment
+  datasource:
+    url: jdbc:postgresql://${PGHOST:localhost}:${PORT:5433}/postgres
+    username: postgres
+    password: postgres
+  liquibase:
+    change-log: classpath:db/changelog/changelog-master.xml
+logging:
```

Comment:
DEBUG log level is not good option for default profile, create dev profile for such purpose.